src/currentprocess/terminalsource.rs

```diff
@@ -9,9 +9,10 @@ use std::{
 pub(crate) use termcolor::Color;
 use termcolor::{ColorChoice, ColorSpec, StandardStream, StandardStreamLock, WriteColor};
 
+use super::{process, varsource::VarSource};
```

Comment:
Please keep this import grouped with the other super:: import

Reply:
Done.